inst/extdata/install-packages.R

```diff
@@ -35,74 +37,51 @@ installationDeps <- function(installOption) {
       )
     }
   }
-
-  # install.packages("pkgbuild", quiet = TRUE)
-  # install.packages("cli", quiet = TRUE)
-  # install.packages("remotes", quiet = TRUE)
-  # install.packages("rstudioapi", quiet = TRUE)
-
-  # require("pkgbuild")
-  # require("cli")
-  # require("remotes")
-  # require("rstudioapi")
+  cli_alert_success("Install prerequisite dependencies")
 }
 
-# check if rtools is installed
-checkRtools <- function() {
+check_RTools <- function() {
+  cli_progress_step("Verify RTools installation")
   if (!pkgbuild::find_rtools()) {
     cli_alert_danger("Rtools is not installed. Please install it before continuing.")
     cli_alert_info("You can download Rtools from https://cran.r-project.org/bin/windows/Rtools/")
     cli_alert_info("Please install the latest version of Rtools.")
     cli_alert_info("After installing Rtools, please restart R and run this script again.")
     stop("Rtools is not installed.")
   }
+  cli_progress_done(result = "done")
 }
 
-# Get PKSim minimal
-getPKSimMinimal <- function() {
-  download.file("https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/pksim_minimal.zip",
-                destfile = "pksim_minimal.zip"
-  )
-  unzip("pksim_minimal.zip", exdir = "PKSim")
-  file.remove("pksim_minimal.zip")
-}
+get_PKSim_Minimal <- function(install_option) {
 
-# Run test simulation
-runTestSimulation <- function() {
-
-  require(esqlabsR)
-  projectConfiguration <- createDefaultProjectConfiguration()
-  # Define which scenarios to run
-  scenarioNames <- c("TestScenario")
-  # Set scenario names to NULL if you want to simulate all scenarios defined in the
-  # excel file
-  # scenarioNames <- NULL
-
-  # Create `ScenarioConfiguration` objects from excel files
-  scenarioConfigurations <- readScenarioConfigurationFromExcel(
-    scenarioNames = scenarioNames,
-    projectConfiguration = projectConfiguration
-  )
+  # Only for local installation
+  if (install_option == 1) {
 
-  scenarios <- createScenarios(scenarioConfigurations)
+    cli_progress_step("Get minimal version of PKSim.")
 
-  simulatedScenarios <- runScenarios(scenarios = scenarios)
+    download.file("https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-r/artifacts/pksim_minimal.zip",
+                  destfile = "pksim_minimal.zip"
+    )
+    unzip("pksim_minimal.zip", exdir = "PKSim")
+    file.remove("pksim_minimal.zip")
 
-}
+    cli_progress_step("Verify PKSim installation")
+    # test if PK-Sim.R dll can be loaded
+    initPKSim()
```

Comment:
This will not work here because `ospsuite`  has not been installed yet.

Reply:
switched order in the main function "setup_esqlabsR"